.pre-commit-config.yaml

```diff
@@ -0,0 +1,9 @@
+repos:
```

Comment:
Two hooks have been added for pre-commit.  Prettier, to keep with our existing standards, and a large file check.  The large file check wasn't necessary by events, just seemed like a good idea.